distribution/lib/Standard/Database/0.0.0-dev/src/Internal/IR/SQL_Expression.enso

```diff
@@ -103,6 +103,14 @@ type SQL_Expression
          enclosing `Let` value.
     Let_Ref (name : Text) (binder : Text) (standalone_expression : SQL_Expression)
 
+    ## PRIVATE
+       The internal representation of an  list of SQL expressions which will be
+       separated by commas in a list in the final SQL query.
+
+       Arguments:
+       - value: the values thatt should be built into a comma separated list.
+    List values:Vector
```

Comment:
I'm feeling like we need to be careful about `SQL_Expression` extensions. Normally the idea was that this type was always representing some SQL expression valid in the proper context. But the `SQL_Expression.List` is not really valid on its own, it is only valid in some very specific positions inside of `SQL_Expression.Operation`.

OTOH I don't have much better suggestions for alternative solutions, I guess it's not worse than some ugly tricks for passing variable lists of arguments I've used in a few operations.

Maybe let's added a comment here that it is only valid in some places and needs to be used carefully?

Reply:
Done